chrome_extension/popup.js

```diff
@@ -0,0 +1,58 @@
+let category = null
+
+const shareButton = document.querySelector('#submit')
+const allButtons = document.querySelectorAll('.button')
+const jsConfetti = new JSConfetti()
+
+allButtons.forEach((button) => {
+  button.onclick = () => {
+    allButtons.forEach((button) => button.classList.remove('selected'))
+    button.classList.add('selected')
+    category = button.id
+    shareButton.style.display = 'flex'
+    explode()
```

Comment:
```suggestion
```
Can we remove the explode here? I love the confetti, but I think it should only happen after the submit. Wdyt?